node-manager/src/socket.rs

```diff
@@ -4,15 +4,23 @@ use crate::utils;
 use crossbeam_channel::{unbounded, Receiver, Sender};
 use futures::lock::Mutex;
 use futures::{SinkExt, StreamExt};
+use gloo_net::websocket::events::CloseEvent;
 use gloo_net::websocket::Message;
-use lightning::ln::peer_handler;
-use log::{debug, error, info, trace, warn};
+use lightning::ln::peer_handler::{self, SocketDescriptor};
+use log::{debug, error, info, trace};
 use std::collections::HashMap;
 use std::hash::Hash;
 use std::sync::atomic::{AtomicBool, AtomicU64, Ordering};
 use std::sync::Arc;
 use wasm_bindgen_futures::spawn_local;
 
+use serde::{Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize)]
+enum MutinyProxyCommand {
+    Disconnect { to: Vec<u8>, from: Vec<u8> },
```

Comment:
I think for this json it will be `{"to": [1,2,1], "from": [23,4,2]}` so we could have some potential overlap, might be worth adding a tag to this so we know which command and don't have potential overlap